Documentation.docc/Purchases.md

```diff
@@ -0,0 +1,110 @@
+# ``RevenueCat/Purchases``
+
+@Metadata {
+    @DocumentationExtension(mergeBehavior: override)
+}
```

Comment:
This tells Xcode to use the overview from here and not the one in the class declaration in the `Purchases.swift` file. That one was a little too noisy for an overview in a dedicated document.